### ISSUE TITLE
where is the softmax?

### ISSUE THREAD

**Issue by alchemistwu** [kratzert/finetune_alexnet_with_tensorflow#12]
It seems that 

> tf.nn.softmax_cross_entropy_with_logits

 needs outputs to be a softmax,right?
I'm  just curious about that,no offence

**Comment by kratzert** [comment]
Well than have a look at the official TF documentation of [tf.nn.softmax_cross_entropy_with_logits](https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits).

> WARNING: This op expects unscaled logits, since it performs a softmax on logits internally for efficiency. Do not call this op with the output of softmax, as it will produce incorrect results.

So no, what this function expects is the unscaled output of the network, not the softmax. 

Hope I could clear your doubts.

**Comment by nAjAcAt** [comment]
Thanks a lot! lol
